Mathlib/SetTheory/Cardinal/Divisibility.lean

```diff
@@ -61,7 +61,7 @@ theorem isUnit_iff : IsUnit a ↔ a = 1 := by
     exact zero_ne_one ht
 #align cardinal.is_unit_iff Cardinal.isUnit_iff
 
-instance : Unique Cardinal.{u}ˣ where
+noncomputable instance : Unique Cardinal.{u}ˣ where
```

Comment:
I don't know why this happened nor whether I should be worried.

Reply:
It's replaceable with:
```
instance uniqueCardinalUnits : Unique Cardinal.{u}ˣ where
  default := ⟨1, 1, by simp, by simp⟩ 
  uniq a := Units.val_eq_one.mp <| isUnit_iff.mp a.isUnit
```
The first `CommMonoid` instance it can find on `Cardinal` comes via a noncomputable `LinearOrderedCommMonoid`.

You can avoid this by instead using
```
instance commMonoid : CommMonoid Cardinal.{u} := CommSemiring.toCommMonoid
```
to put in a higher priority instance. I'm not sure where it would be best to put this.